charts/loki-simple-scalable/values.yaml

```diff
@@ -172,7 +172,7 @@ prometheusRule:
 # Configuration for the write
 write:
   # -- Number of replicas for the write
-  replicas: 1
+  replicas: 3
```

Comment:
Could you explain why autoscaling does not fulfil most common use? Or just your personal local environment require so?

Reply:
I think it's better to just comment out the autoscaling value, otherwise how does the new users know where to go when they need those syntax? e.g. min 3 and max 7 in some big / wide cluster.

Reply:
@zanhsieh I'm confused by this comment. I don't see anywhere in the chart that the auto-scaling value is being used? I don't think the `write` pod ever had this section, and I think that section for `read` was just copied over by accident when deriving this chart from `loki-distributed`. I think the chart would need a `HorizontalPodAutoscaler` in order to enable auto-scaling. Instead of adding that, it seems a lot simpler to me to just bump the replicas from 1 to 3 so it works with the default replication factor of 3. WDYT?

Reply:
@trevorwhitney 
Could your own case represent 75% of the existing loki-simple-scalable user cases? Could you prove it? 
If the syntax for autoscaling lost and one day the existing user needs those, what should they do? Go back to this commit history and dig it up what should it be? Are you sure this public repo would keep all pieces of history?

Reply:
@zanhsieh sorry for the confusion, but I don't understand your question? The simple scalable helm chart has never used autoscaling, so I'm not sure what history you're referring to? Removing this config will work with 100% of `loki-simple-scalable` deployments that already exist. If someone needs to find this syntax, they can look at the `loki-distributed` chart which includes an autoscaling configuration as well as templates for `HorizontalPodAutoscaler`.

As for the replica change, the driving force behind SSD mode is simplicity, which is the motivation for not including autoscaling and for including defaults that work with Loki defaults as much as possible. If someone would prefer using 1 replica, those values are easy to change, and they can look at the included `ci` configurations for examples on how to do that.